asteroid/losses/pit_wrapper.py

```diff
@@ -191,6 +221,169 @@ def best_perm_from_perm_avg_loss(loss_func, est_targets, targets, **kwargs):
         batch_indices = torch.stack([perms[m] for m in min_loss_idx], dim=0)
         return min_loss, batch_indices
 
+     @staticmethod
+    
+    @staticmethod
+    def best_part_mix_it(loss_func, est_targets, targets, **kwargs):
+        """ Find best partition of the estimated sources that gives
+            the minimum loss for the MixIT training paradigm in [1].
+             Valid for any number of mixtures as soon as they contain
+             the same number of sources.
+
+        Args:
+            loss_func: function with signature (targets, est_targets, **kwargs)
+                The loss function batch losses from.
+            est_targets: torch.Tensor. Expected shape [batch, nsrc, *].
+                The batch of target estimates.
+            targets: torch.Tensor. Expected shape [batch, nsrc, *].
+                The batch of training targets.
+            **kwargs: additional keyword argument that will be passed to the
+                loss function.
+
+        Returns:
+            tuple:
+                :class:`torch.Tensor`: The loss corresponding to the best
+                permutation of size (batch,).
+
+                :class:`torch.LongTensor`: The indexes of the best permutations.
+
+        References:
+            [1] Scott Wisdom and Efthymios Tzinis and Hakan Erdogan and Ron J Weiss
+            and Kevin Wilson and John R Hershey, "Unsupervised sound separation using
+            mixtures of mixtures." arXiv preprint arXiv:2006.12701 (2020) $
+        """
+
+        # check input dimensions
+        assert est_targets.shape[0] == targets.shape[0]
+        assert est_targets.shape[2] == targets.shape[2]
+
+        # get L, N and M
+        L = targets.shape[1]        # number of mixtures 
+        N = est_targets.shape[1]    # number of estimated sources
+        if N % L != 0:
+            raise ValueError('The mixtures are assumed to contain the same number of sources')
+        M = N // L                  # number of sources in each mixture
```

Comment:
We try to avoid variable names like that. 
Using `n_mixtures` or `n_est` would be better.